src/components/AddTransaction/index.js

```diff
@@ -1,26 +1,50 @@
-import {useState} from 'react';
+import {useEffect, useState} from 'react';
 import {useParams} from 'react-router-dom';
 
 import useStore from '../../hooks/useStore';
 import {Form} from '../AddNewAccount/styled';
 
-export default function AddTranscation() {
+export default function AddTransaction() {
 	const [transaction, setTransaction] = useState({
 		date: '',
 		change: '',
 		note: '',
 		category: '',
```

Comment:
Maybe add a new variable "initialTransaction" which refers to your initial state. This way you can use the variable later as well and won't repeat yourself. Just a nice and quick way to reset a state 🙂

Reply:
For example you can use the variable in the set Transaction function at line 48 at the index.js.

Reply:
Oh! Yeah that makes total sense.. 😂 My thoughts were way more complicated than what it actually is. Keep it simple! Thank you.

Reply:
👍🏻😃